gui/web/src/kelp-ops-api/serverMetadata.js

```diff
@@ -1,5 +1,5 @@
 export default (baseUrl) => {
-    return fetch(baseUrl + "/api/v1/serverMetadata").then(resp => {
+    return fetch(baseUrl + "/api/v1/serverMetadata", {method: "GET"}).then(resp => {
```

Comment:
so the thing with interceptor is, it overrides the header on every request, so when we make a fetch request it looks for header, but in this case, it was unable to find it so I deliberately added {method: "GET"} in fetch request to have a valid header to override.

Reply:
Ran into this issue while running in kaas mode in my local copy.
`/newSecretKey` (`newSecretKey.js`) fails in the interceptor because this `config` isn't specified. Ideally we shouldn't have to specify this since it's optional in the `fetch` API which defaults to using a GET method.

Can we try by adding a null check to `config` in the interceptor rather than just checking if `config.headers` is null?